packages/headless/src/controllers/commerce/search/headless-search.ts

```diff
@@ -29,6 +29,7 @@ import {
 export interface Search extends Controller, SearchAndListingSubControllers {
   /**
    * Executes the first search.
+   * @param analyticsEvent - The analytics event to log.
```

Comment:
We shouldn't have that param at all on the executeFirstSearch method of the commerce search contoller; it makes no sense because legacy analytics are not supported at all in the commerce sub-package.